pvlib/iotools/util.py

```diff
@@ -0,0 +1,42 @@
+"""
+Common functions that support the different data readers / writers
+
+TODO: add a sanity checker
+* is the data between sunrise & sunset --> avoid timezone / timeshift errors
+
+"""
+
+
+import pytz
+# How to get a time zone from a location using 
+# latitude and longitude coordinates?
+# http://stackoverflow.com/a/16086964
+# upstream: https://github.com/MrMinimal64/timezonefinder
+from timezonefinder import TimezoneFinder
+
+def get_loc_latlon(lat, lon):
+    """Returns the timezone for a coordinate pair.
+    """
+    
+    
+    tf = TimezoneFinder()
+    tz = tf.timezone_at(lng=lon, lat=lat)
+    
+    return tz
+    
+def localise_df(df_notz, tz_source_str='UTC', tz_target_str=None):
```

Comment:
Why do we need this function? Seems more clear and robust to only call the pandas methods.